src/en/mobile/android/development/general-navigation.md

```diff
@@ -0,0 +1,179 @@
+﻿---
+title: "General navigation"
+---
+
+# General navigation
+
+## Have sufficiently large click areas
+
+**Target&nbsp;:** everyone and in particular for people with motor disabilities.  
+**When&nbsp;:** from the design phase and during development.
+
+**Description&nbsp;:**  
+
+Insufficient size for a clickable element can make it difficult to interact with the application. Each clickable element of the application must be of sufficient size.
+
+**To be verified&nbsp;:**
+
+- The click zone has a minimum size of 48 dp.
+- The click zone has a margin around the element of at least 8 dp
+- Click errors can be undone by moving your finger out of the clickable area before releasing.
+
+**Valid examples&nbsp;:** 
+
+Classic example &nbsp;:
+<pre><code class="xml">&lt;Button 
+    android:text="Click"
+    android:layout_width="48dp"
+    android:layout_height="48dp"
+    android:margin="8dp"
+&#47;&gt;</code></pre>
+
+Padding example &nbsp;:
+<pre><code class="xml">&lt;Button 
+    android:text="Click"
+    android:layout_width="40dp"
+    android:layout_height="40dp"
+    android:padding="4dp"
+    android:margin="8dp"
+&#47;&gt;</code></pre>
+
+**Invalid examples&nbsp;:** 
+
+<pre><code class="xml">&lt;Button 
+    android:text="Click"
+    android:layout_width="48dp"
+    android:layout_height="48dp"
+    android:margin="4dp"
+&#47;&gt;</code></pre>
+
+
+<pre><code class="xml">&lt;Button 
+    android:text="Click"
+    android:layout_width="40dp"
+    android:layout_height="40dp"
+    android:margin="8dp"
+&#47;&gt;</code></pre>
+
+
+**Reference <abbr>WCAG</abbr>&nbsp;:**  
+- <a lang="en" href="https://www.w3.org/TR/WCAG21/#target-size">2.5.5 Target Size</a>
+- <a lang="en" href="https://www.w3.org/TR/WCAG21/#pointer-cancellation">2.5.2 Pointer Cancellation</a>
+
+
+<br/><br/>
+## Have readable texts
+
+**Target&nbsp;:** everyone and in particular people with visual impairments and dyslexics.  
+**When&nbsp;:** from the design phase and during development.
+
+**Description&nbsp;:**
+
+The text can cause readability problems when it is too small or the font has impasto : a minimum text size of 14sp is recommended. 
```

Comment:
s/impasto/serif/
[NB: impasto is _empâtement_, it's a painting method, while serif is _empattement_, the typography thing 😉 ]